.agent/workflows/all.md

```
@@ -4,17 +4,45 @@ description: Load all project skills by calling other workflows
-   - `/basic`
+> **Estimated context: ~42K tokens** (29 skills)
 
-2. Load overview skills
-   - `/overview`
+Read all skills:
 
-3. Load DRN framework skills
-   - `/drn`
+**Basic:**
+   - `view_file .agent/skills/basic-agentic-development/SKILL.md`
+   - `view_file .agent/skills/basic-documentation/SKILL.md`
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)